common.go

```diff
@@ -48,6 +48,167 @@ const (
 	KAOHI_DEFAULT_SYSLOG_PROTO       = "tcp"
 )
 
+const (
+	KAOHI_HCL_OPTIONS = `cmdline "log_directory" {
```

Comment:
@jjr840430 can you extract this out to a separate file, called config.mel

Reply:
should `config.mel` be located in the same directory with executable?

Reply:
@ronaldtse Fixed, I added `config.mel` in `./bin` directory.

Reply:
@jjr840430 not really -- `config.mel` should only be available in compilation, the user doesn't need it, so parse and store the contents of the file into the executable.

Reply:
when I included `config.mel` to be compiled with `.go` files, the compiler said:

```
named files must be .go files
```

Reply:
There must be some way you can pre-parse the mel file into Go, and then compile Kaohi?